doc/articles/interop/wasm-javascript-2.md

```diff
@@ -279,7 +255,7 @@ Let's create an app to integrate a Syntax Highlighter named [`PrismJS`](https://
 This sample is a very simple integration as there is no _callback_ from HTML to managed code and _PrismJS_ is a self-contained framework (it does not download any other JavaScript dependencies).
 Some additional improvements can be done to make the code more production ready:
 
-* **Make the control multi-platform**. A simple way would be to use a WebView on other platforms, giving the exact same text-rendering framework everywhere. The code of this sample won't compile on other targets.
-* **Create script files instead of generating dynamic JavaScript**. That would have the advantage of improving performance and make it easier to debug the code. A few projects are also using TypeScript to generate JavaScript. This approach is done by Uno itself for the [`Uno.UI.Wasm`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/Uno.UI.Wasm.csproj) project.
+* **Make the control multi-platform**. An other would be to use a [WebView](xref:Uno.Controls.WebView2) on other platforms, giving the exact same text-rendering framework everywhere. The code of this sample won't run on non-WebAssembly targets.
+* **Use script files instead of using `ExecuteJavascript`**. That would have the advantage of improving performance and make it easier to debug the code.
```

Comment:
```suggestion
* **Make the control multi-platform**. Another would be to use a [WebView](xref:Uno.Controls.WebView2) on other platforms, giving the exact same text-rendering framework everywhere. The code of this sample won't run on non-WebAssembly targets.
* **Use script files instead of using `ExecuteJavascript`**. That would have the advantage of improving performance and making it easier to debug the code.
```